docs/guides/interactive-components.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: default
+title: "Interactive Components"
+lang: en
+---
+
+# Interactive Components
+
+[Interactive components](https://api.slack.com/reference/block-kit/interactive-components) are a subset of Block Kit elements that add interactivity to various app surfaces. Interactions on blocks may happen not only in messages but also on [Modals]({{ site.url | append: site.baseurl }}/guides/modals), and [Home tabs]({{ site.url | append: site.baseurl }}/guides/app-home).
+
+### Slack App Configuration
+
+To enable Interactive Components, visit the [Slack App configuration page](http://api.slack.com/apps), choose the app you're working on, and go to **Features** > **Interactive Components** on the left pain. There are three things to do on the page.
+
+* Turn on the feature
+* Set the **Request URL** to `https://{your app's public URL domain}/slack/events`
+* Click the **Save Changes** button at the bottom for sure
+
+### What Your Bolt App Does
+
+All your app needs to do to handle Slack requests by user interactions are:
+
+1. Verify requests from Slack (read [this](https://api.slack.com/docs/verifying-requests-from-slack) if unfamiliar)
+1. Parse the request body and check if the `action_id` in a block is the one you'd like to handle
+1. Build a reply message or surface to interact with the user further
+1. Respond with 200 OK as the aknowledgement
+
+Your app has to respond to the request within 3 seconds by `ack()` method for sure. Otherwise, the user will see the timeout error on Slack.
```

Comment:
Sometimes they'll need to include parameters in `ack()` (for example, with external selects). It would be helpful to notes something about that here.